.github/workflows/php.yml

```diff
@@ -120,30 +90,6 @@ jobs:
         with:
           version: 'v1.22.6'
 
-      # Export the build tag in the deployment stage as well as build because it's difficult to reconcile the image
-      # build tag from build.matrix1.output vs build.matrix2.output.
-      # If we receive a container name
-      # TODO: Deprecate when on helm
-      - name: "Export build tag"
-        id: export_build_tag_container_provided
-        if: github.event.inputs.CONTAINER != ''
-        env:
-          BUILD: ${{ github.run_number }}
-        run: |
-          echo "build_num=$(($BUILD+10000))" >> $GITHUB_OUTPUT
-          echo "image_tag_build=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))" >> $GITHUB_OUTPUT
-          echo "image_tag_branch=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}" >> $GITHUB_OUTPUT
-      # If we do not receive a container name, use the repo name
-      - name: "Export build tag"
-        id: export_build_tag
```

Comment:
It looks like this specific step is required as part of the helm deploy step tag:

```
tag: ${{ steps.export_build_tag.outputs.build_num }}
```

Reply:
yey for copy+pasta :polite-cat:

Reply:
I mean, given that moving to helm only requires we have the build number (+1000...) and not the full image tag. We could set an output variable in build which takes either number output from either matrix, then passes that on....Might be worth a try in another branch?

Reply:
Yeah, might be worth a try if it cleans things up

Reply:
Would even be worth exporting to a re-usable action....Will add a ticket